src/appkit.cs

```diff
@@ -760,6 +760,29 @@ interface NSApplication : NSAccessibilityElementProtocol, NSAccessibility {
 		void EnumerateWindows (NSWindowListOptions options, NSApplicationEnumerateWindowsHandler block);
 	}
 
+	[Static]
+	interface NSAboutPanelOption {
```

Comment:
This looks like could be turned into a [SmartEnum](https://github.com/xamarin/xamarin-macios/blob/master/docs/website/binding_types_reference_guide.md#enum-attributes)